react-components/src/components/RevealToolbar/LayersButton.tsx

```diff
@@ -131,9 +148,15 @@ export const LayersButton = (): ReactElement => {
       isToggled: boolean;
       isActive: boolean;
     } {
+      let visible = true;
+      image360Layers?.forEach((layer) => {
+        if (layer.siteId === image360Collection.id) {
+          visible = layer.applied;
+        }
+      });
```

Comment:
Similar to other comment related to `forEach`, can't you just use `.find` and then you don't need to define another variable and can use it in `isToggled`.